sympy/combinatorics/free_groups.py

```diff
@@ -230,14 +231,40 @@ def index(self, gen):
             raise ValueError("expected a generator of Free Group %s, got %s" % (self, gen))
 
     def order(self):
-        """Returns the order of the free group."""
+        """Returns the order of the free group.
+
+        Examples
+        ========
+
+        >>> from sympy.combinatorics.free_groups import free_group
+        >>> F, x, y = free_group("x, y")
+        >>> F.order()
+        oo
+
+        >>> from sympy.combinatorics.free_groups import free_group
+        >>> F, x, y = free_group("x, y")
+        >>> (x ** 0).order()
```

Comment:
This is not an example of `FreeGroup`'s `order` method. `x**0` is an instance of `FreeGroupElement`. You could add this as an example to the relevant method in the `FreeGroupElement` class if you want.

An example of a free group of order 1 would be `free_group("")[0]`.

Reply:
OK. I will add this as an additional example in the ```FreeGroupElement``` class and fix the existing one to include an appropriate information.